packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx

```diff
@@ -46,7 +46,8 @@ function CategoryList({
           ...(!embedded && { maxHeight: 175 }),
         }}
       >
-        {items.map((item, idx) => {
+        {items.filter(item => !item.hidden)
+              .map((item, idx) => {
```

Comment:
I don't really like having this on a second line, but the linting rules said it was needed.